README.md

```diff
@@ -16,7 +16,7 @@
 
 <!-- /ix-docs-ignore -->
 
-<!-- NB: Run `npx markdown-toc README.md -i --maxdepth 4` to generate TOC :) -->
+<!-- NB: Run `npx markdown-toc README.md -i --maxdepth 4` to generate TOC, then manually adjust indentation to 4 spaces :) -->
```

Comment:
No change needed, but this command will also do the same thing without us having to manually re-indent every time: `npx markdown-toc README.md --maxdepth 4 | sed -e 's/[[:space:]]\{2\}/    /g'`. All we have to do is manually insert the TOC, it won't insert it automatically anymore.


```suggestion
<!-- NB: Run `npx markdown-toc README.md --maxdepth 4 | sed -e 's/[[:space:]]\{2\}/    /g'` to generate TOC, and copy the result from the Terminal to replace the TOC below :) -->
```